src/puppetlabs/puppetdb/anonymizer.clj

```diff
@@ -139,6 +139,8 @@
        :type (random-type-name)
        :title (random-string (max 10 (count value)))
        :parameter-name (random-string-alpha (max 10 (count value)))
+       :catalog-input-type (random-string-alpha (max 10 (count value)))
```

Comment:
How do we feel about not anonymizing types? It will be one of three things `hiera`, `function`, `template`.

Reply:
Nevermind, this only happens for `full` 👍 makes sense. Ignore me.